src/libultra/gu/sins.c

```diff
@@ -1,5 +1,6 @@
 #include "ultra64.h"
-#include "sintable.c"
+
+#include "sintable.inc.c"
```

Comment:
This should be `sintable.h` anyway. And if you do it for sins, might as well do the same for atan.

Reply:
It's a file that contains data so I'm not really following why it should be a `.h`

As for atan do you mean that `sys_math_atan.c` should have its table split as well? I didn't work on any of these files so I don't know why they were handled differently, but I assume there was a reason

Reply:
That's how it is in libultra: https://github.com/decompals/ultralib/blob/main/src/gu/sins.c

Yes, the atan table in sys_math_atan. The only difference really seems to be that it's a game library file rather than a libultra one.

Reply:
(This is definitely correct since the source is on the CDs.)

Reply:
I see, well I don't think it's right to make it a `.h` even if that's how it was originally, but I'm fine with putting the table in the file itself like it is for sys_math_atan (or we can make a separate c file for the atan table if we prefer)

Reply:
Even if the original table was a `.h` I think we shouldn't put it on a `.h`.
I think putting it on a file itself would be better tho

Reply:
I vote to let Tharo move it to a .h eventually in his relentless efforts to clean-up libultra

Reply:
@tharo ultralib wen

Reply:
As usual my thoughts are that for libultra we should try to be as compliant as possible, however I think in this case we can leave it for later